src/Login/Login.jsx

```diff
@@ -35,9 +36,10 @@ const Login = ({ history }) => {
   return (
     <Card id="registration-card">
       <div id="feed">
+        <img src={logo} alt="Cannot load " />
```

Comment:
Als should give a descriptive name for name for when the image goes missing e.g. "Tindoori Logo"